richtextfx/src/integrationTest/resources/org/fxmisc/richtext/api/FxmlTest.fxml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.layout.VBox?>
+<?import org.fxmisc.richtext.StyleClassedTextArea?>
+<?import javafx.scene.control.ContextMenu?>
+<?import javafx.scene.control.MenuItem?>
+
+<VBox xmlns:fx="http://javafx.com/fxml/1">
+    <StyleClassedTextArea editable="true" wrapText="true" autoScrollOnDragDesired="true" />
+    <StyleClassedTextArea contextMenuXOffset="12.0" contextMenuYOffset="34.0" >

```

Comment:
These two sets of properties should be combined into the same area

Reply:
I can do that, I did it this way because I thought its easier to read than having one long line of properties :-)

Reply:
Does it need to be a one-liner? Could you do something like
````xml
<StyleClassedTextArea
    editable="true"
    wrapText=true"
    ... 
    <contextMenu>...</contextMenu>
</StyleClassedTextArea>